src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -425,6 +425,11 @@ public sealed partial class SqlCommand : System.Data.Common.DbCommand, System.IC
         public System.IAsyncResult BeginExecuteXmlReader() { throw null; }
         public System.IAsyncResult BeginExecuteXmlReader(System.AsyncCallback callback, object stateObject) { throw null; }
         public System.Xml.XmlReader EndExecuteXmlReader(System.IAsyncResult asyncResult) { throw null; }
+        public System.IAsyncResult BeginExecuteReader() { throw null; }
```

Comment:
@galakt These should be added to System.Data.SqlClient.NetCoreApp.cs and not the System.Data.SqlClient.cs file. 
These new APIs should be exposed in netcoreapp specific build. Netstandard cannot be updated.

Reply:
Thanks @galakt 
@danmosemsft My understanding is that any new APIs added even to existing types, should be moved to netcoreapp specific builds based on the conversations we had on this thread. https://github.com/dotnet/corefx/pull/32245#pullrequestreview-154446803 

Is this assumption correct? I guess we will need to update the baseline files to represent this.

Reply:
@danmosemsft Can we finally resolve question about place for my Refs please?

Reply:
I'm not sure we have been consistent - @ericstj?

Reply:
@karelz Sorry, can you push this?

Reply:
@saurabh500 @ericstj @danmosemsft Any updates on this?

Reply:
SqlClient is not part of netstandard, so its APIs are not pinned by being in netstandard.  You do however, need to consider support.  If you add API to the reference assembly you should be sure it is supported everywhere that API can be used.  So if you're exposing it in a netstandard2.0 reference assembly this means it needs to be supported in .NET 4.6.1 and later, .NETCoreApp 2.0 or later, UAP, etc (see https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support).  For UAP and NETCoreApp you can simply update the implementation assembly that is in the package, desktop is the limiting concern here since you cannot update that (and perhaps mono/Xamarin).

Reply:
@saurabh500 If i understood correct, i should revert "move to NetCoreApp.cs" commit. Am i right?

Reply:
@galakt Yes that is right. 
@ericstj  Thanks for the clarification.